.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Report a Bug
+about: Found an issue? Let us fix it.
+---
+
+Please ensure you do the following when reporting a bug:
+
+- [ ] Provide a concise description of what the bug is.
+- [ ] Provide information about your environment.
+- [ ] Provide clear steps to reproduce the bug.
+- [ ] Attach applicable logs. Please do not attach screenshots showing logs unless you are unable to copy and paste the log data.
+- [ ] Ensure any code / output examples are [properly formatted](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code) for legibility.
+
+Note that some logs needed to troubleshoot may be found in the `/pgdata/<CLUSTERNAME>/pg_log` directory on your Postgres instance.
+
+An incomplete bug report can lead to delays in resolving the issue or the closing of a ticket, so please be as detailed as possible.
+
+If you are looking for [general support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/), please view the [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) page for where you can ask questions.
+
+Thanks for reporting the issue, we're looking forward to helping you!
+
+## Overview
+
+Add a concise description of what the bug is.
+
+## Environment
+
+Please provide the following details:
+
+- Platform: (`Kubernetes`, `OpenShift`, `Rancher`, `GKE`, `EKS`, `AKS` etc.)
+- Platform Version: (e.g. `1.20.3`, `4.7.1`)
+- PGO Image Tag: (e.g. `centos8-4.7.1`)
+- Postgres Version (e.g. `13`)
+- Storage: (e.g. `hostpath`, `nfs`, or the name of your storage class)
+
+## Steps to Reproduce
+
+### REPRO
+
+Provide steps to get to the error condition:
+
```

Comment:
Why do we need this file?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81